neo/Network/P2P/Peer.cs

```diff
@@ -57,16 +61,18 @@ protected void AddPeers(IEnumerable<IPEndPoint> peers)
             }
         }
 
-        protected void ConnectToPeer(IPEndPoint endPoint)
+        protected void ConnectToPeer(IPEndPoint endPoint, bool isTrusted = false)
         {
             endPoint = endPoint.Unmap();
             if (endPoint.Port == ListenerPort && localAddresses.Contains(endPoint.Address)) return;
+
+            if (isTrusted) TrustedIpAddresses.Add(endPoint.Address);
```

Comment:
Never is removed a Trusted IpAddress ? how is defined a trusted Ip?

Reply:
If a particular operator is running multiple nodes they can add IPs of their other nodes as trusted for example. By default that flag is not set when connecting to peers during the normal mechanism.

Reply:
Also, as you can see from the code trusted IPs aren’t subject to being rejected due to max connections; although they still limit to 3 connections per IP.

Reply:
So at the moment no Address is Trusted? neither from `protocol.json`?